### DISCUSSION TITLE
Transparent & system accent Spotify is coming...

### DISCUSSION THREAD

**Discussion by SunsetTechuila** [nimsandu/spicetify-bloom#270] [Show and tell]
...someday :)

![image](https://github.com/nimsandu/spicetify-bloom/assets/115353812/ea47d6c7-ec33-4bce-9855-e3430876f023)

**Comment by SunsetTechuila** [comment]
Should it be optional? What do you think @nimsandu @windowz414?

**Reply by SunsetTechuila** [reply]
came to that solution - the existing color schemes will remain as they are, but fluent_light and fluent_dark will be added, and transparency effects will be applied only to them

**Reply by SunsetTechuila** [reply]
maybe a bit messy but gives freedom to the user

**Comment by SunsetTechuila** [comment]
![image](https://github.com/nimsandu/spicetify-bloom/assets/115353812/d255ecf7-9338-4d67-a640-a85cb963d5bf)
![image](https://github.com/nimsandu/spicetify-bloom/assets/115353812/b46dd25b-0759-4a3f-ab3f-6f0ea6425a3b)

https://github.com/nimsandu/spicetify-bloom/tree/fluent

Works only with `fluent_dark` color scheme.
Use [`.fluent`](https://github.com/nimsandu/spicetify-bloom/blob/fluent/src/user.css#L2765-L2771) block for configuration.
[Accent color](https://github.com/nimsandu/spicetify-bloom/blob/fluent/src/color.ini#LL164C3) and background image should be set manually.

**Reply by Ergoprett** [reply]
I Just tried it out, and it seems like it's just using a JPEG file as a background in the app and doesn't have info about what's behind the window, so it isn't really transparent.
Is that how It's supposed to work? Or is this a bug?

**Reply by SunsetTechuila** [reply]
ofc transparency is fake, what did you expect? :)
there is no way to make it really transparent using only spicetify

I've even said:
> background image should be set manually

also:
![image](https://github.com/nimsandu/spicetify-bloom/assets/115353812/bc002663-595a-461b-b034-6f6f9ae99316)

**Reply by SunsetTechuila** [reply]
but it looks good enough to me (**very** good imo)

**Reply by Ergoprett** [reply]
Okay didn't know that. 
I really know nothing about window transparency. Just changed the wallpaper looks great as it is. 
Sorry for the inconvenience

**Reply by SunsetTechuila** [reply]
Background and accent colour will be set automatically on release when using powershell installation script

**Comment by SunsetTechuila** [comment]
LibraryX support is here!

![image](https://github.com/nimsandu/spicetify-bloom/assets/115353812/a1c64155-3cbb-4ab6-9724-58d9d24ad0e1)
![image](https://github.com/nimsandu/spicetify-bloom/assets/115353812/a2eebcf8-7080-45b1-963a-f79ece551671)

**Comment by nimsandu** [comment]
@SunsetTechuila hey. i also tested it out! wow mate it looks stunning. well @Ergoprett in windows 11 mica material is not bluring the content behind it anyways. it just blurs the background!

**Comment by nimsandu** [comment]
![image](https://github.com/nimsandu/spicetify-bloom/assets/57834844/2b1a8e39-549b-44a1-bcca-22d212715afd)
@SunsetTechuila @windowz414

**Reply by SunsetTechuila** [reply]
Better to use with classic UI right now

**Comment by nimsandu** [comment]
@SunsetTechuila @windowz414 after releasing this great feature someday, we will change the whole readme by adding all new screenshots and aligning them in a grid. ill do that. @SunsetTechuila @windowz414 thanks a lot for making this theme better. i have something to tell you both. we will get into that in the near future. 

_@SunsetTechuila dm me through twitter if you can_
